src/components/plugin/definition.scss

```diff
@@ -1,4 +1,19 @@
+.disableSelect {
+  user-select: none;
+}
 .icons {
   display: inline-block;
   vertical-align: middle;
 }
+.imageContainer {
```

Comment:
NOTE: this class also exists in `glossary-popup.scss` but it was not being used as the `definition.tsx` file was using classes in `definition.scss`.